generators/app/tools.json

```diff
@@ -2,14 +2,52 @@
   "auth0": {
     "name": "Auth0 integration",
     "templates": [
+      {
+        "name": "auth0MiddlewareRoutes",
+        "templatePath": "example_routes.ejs",
+        "recommendedPath": "./app/example_routes.js",
+        "conditionalPrompt": "example_files"
+      },
+      {
+        "name": "auth0MiddlewareConfig",
+        "templatePath": "example_index.js",
+        "recommendedPath": "./app/config/example_index.js",
+        "conditionalPrompt": "example_files"
```

Comment:
Change the [tool schema](https://github.com/picollomartin/generator-node-utils/blob/master/test/utils/schemas/tool_schema.json)